scala/src/main/scala/smile/clustering/distances/Binary/HammingDistance.scala

```diff
@@ -0,0 +1,15 @@
+package smile.clustering.distances
+
+class HammingDistance extends Binary_Distance
+{
```

Comment:
Please move `{` up. Please see out scala code for general format guideline. In general, we follow the standard scala library formatting.

Reply:
Harmonized